events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+                '&rsquo;': '"',
+                '&ldquo;': '"',
+                '&rdquo;': '"',
+                '&eacute;': 'e',
+                '&prime;': '\'',
+                '&Prime': '"'
+            });
+        }
+    
+        function addCharacterEntities(newEntities) {
+            var charKeys = [],
+                entityKeys = [],
+                key, encodedChar;
+            for (key in newEntities) {
+                encodedChar = newEntities[key];
+                entityToChar[key] = encodedChar;
+                charToEntity[encodedChar] = key;
+                charKeys.push(encodedChar);
+                entityKeys.push(key);
+            }
+            charToEntityRegex = new RegExp('(' + charKeys.join('|') + ')', 'g');
+            entityToCharRegex = new RegExp('(' + entityKeys.join('|') + '|&#[0-9]{1,5};' + ')', 'g');
+        }
+    
+        function htmlEncode(value) {
+            var htmlEncodeReplace = function(match, capture) {
+                return charToEntity[capture];
+            };
+    
+            return (!value) ? value : String(value).replace(charToEntityRegex, htmlEncodeReplace);
+        }
+    
+        function htmlDecode(value) {
+            var htmlDecodeReplaceFn = function(match, capture) {
+                return (capture in entityToChar) ? entityToChar[capture] : String.fromCharCode(parseInt(capture.substr(2), 10));
+            };
+    
+            return (!value) ? value : String(value).replace(entityToCharRegex, htmlDecodeReplaceFn);
+        }
+    
+        resetCharacterEntities();
+    
+        return {
+            htmlEncode: htmlEncode,
+            htmlDecode: htmlDecode
+        };
+    })();
+
+    function begin() {
+        findTargets();
+        bubbleOn();
+        prizeCalculator("new game");
+        if (!introPlayed) {
+            Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['GAME_INTRO']);
+            introPlayed = true;
+        } else {
+            Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEW_GAME_SFX']);
+        }
+        Entities.callEntityServerMethod(gameZoneProperties.id, "stopConfetti");
+        lights.forEach(function(light) {
+            Entities.callEntityServerMethod(light, "lightsOn");
+        }); 
+    }
+
+    function bubbleOn() {
+        Entities.callEntityServerMethod(bubble, "bubbleOn");
+        Messages.sendMessage(TRIVIA_CHANNEL, JSON.stringify({ 
+            type: "game on" 
+        }));
+        intervalBoard = Script.setInterval(function(){
+            updateAvatarCounter(false);
+            Entities.callEntityServerMethod(prizeDisplay, "textUpdate", [prizeMoney, true]);
+        }, ONE_SECOND_MS);
+    }
+
+    function bubbleOff() {
+        Entities.callEntityServerMethod(bubble, "bubbleOff");
+        Messages.sendMessage(TRIVIA_CHANNEL, JSON.stringify({ type: "game off" }));
+        if (intervalBoard) {
+            Script.clearInterval(intervalBoard);
```

Comment:
set `intervalBoard` to `false` or similar - calling this function won't reset the variable

Reply:
fixed.